### ISSUE TITLE
[Forge] – Servers will not fully start up with Starlight installed

### ISSUE THREAD

**Issue by antibang** [Spottedleaf/Starlight#116]
I apologize if you’re not interested, nor care about Forge 1.16.5 problems, but I figured I’d bring it up anyways on the off chance.

What is strange is that I have what appears to be the exact same issue with the mod RoadRunner. If I install Starlight (or RoadRunner) on the client, it’s completely fine and shows _huge_ performance improvements, considering my modpack’s lag and whatnot is almost entirely due to world gen and lighting! I already reported an issue to RoadRunner's author, but I thought I'd mention I'm having similar issues here if it helps at all ([Here’s my pack](https://www.curseforge.com/minecraft/modpacks/baercraft) if you need a mod list. It’s at the very bottom)

I have 2 servers running my pack. When I install Starlight, the server’s console stops responding, and I’m forced to kill the server. I can’t say if this is the same issue, or even similar, to the problem I have with RoadRunner, but they appear to be extremely similar. It’s just frustrating because Starlight especially, improves performance by _a ton_. My servers, along with anyone else using my pack on a server, would benefit from the mod alone more than all of the other performance mods I have in the pack and on my servers combined.

I have tried both 1.0.0-RC1 and 1.0.0-RC2 versions

**Server log with Starlight 1.0.0-RC2 installed:**
[latest(with).log](https://github.com/PaperMC/Starlight/files/7818277/latest.with.log)

**Server log without Starlight 1.0.0-RC2 installed:**
[latest(without).log](https://github.com/PaperMC/Starlight/files/7818305/latest.without.log)

**Comment by AeiouEnigma** [comment]
I'm on mobile so I can't view your logs atm, but Create and Starlight are incompatible on 1.16, and that's probably the root of your issue.

**Comment by antibang** [comment]
So, I just tested something I should've before reporting, and I'm unsure if this relates to the incompatibility with Create.

If I make a new world on the server, it loads up just fine every time. It just won't load with pre-existing worlds. This is the **same issue** I was having with RoadRunner. I had it installed on my third server I use for events, but it always uses a new world. I added RoadRunner before, and it started up. I guess the same thing with Starlight.
